ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -346,6 +354,11 @@ private void executeStatement(final QueuedCommand queuedCommand) {
       if (closed) {
         LOG.info("Execution aborted as system is closing down");
       } else {
+        if (!rateLimiter.tryAcquire()) {
+          throw new KsqlRestException(
```

Comment:
How will this error get handled?

And can you a unit test to make sure its gets thrown and handled?

Reply:
Yeah writing a unit test rn - it will return an error back up to the user